Sources/QuickAction/Actions/ShowHideFitnessTrailsAction.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ShowHideFitnessTrailsAction.swift
+//  OsmAnd Maps
+//
+//  Created by Max Kojin on 09/08/24.
+//  Copyright © 2024 OsmAnd. All rights reserved.
+//
+
+import Foundation
+
+@objcMembers
+class ShowHideFitnessTrailsAction: BaseRouteQuickAction {
+    
+    static var type: QuickActionType?
```

Comment:
move to base classs

Reply:
if "type" will be moved to base class, it will be inited with the first quick action (cycle routes by example).

But after that on next quick action initing (mtb by example), "type" will get from base class. In will not be nill.  it already will be inited with "cycle routes". So "mtb" and all another quick actions will use "cycle routes". 

<img width="1416" alt="Screenshot 2024-08-09 at 16 04 44" src="https://github.com/user-attachments/assets/4d388bf0-31b5-4219-ac42-b4d7d80ce529">

So we need to use separated variables in each child class. like it was in previous objc realization

<img width="925" alt="Screenshot 2024-08-09 at 16 12 38" src="https://github.com/user-attachments/assets/a7516548-927f-4163-836b-f1cbc63ff99a">